ui/com/user-info.jsx

```diff
@@ -25,8 +25,8 @@ const FLAG_DROPDOWN = [
 class AutoRefreshingComponent extends React.Component {
   constructor(props) {
     super(props)
-    this.state = this.computeState()
-    this.refreshState = props => { this.setState(this.computeState(props)) }
+    this.computeState(null, this.setStateCallback.bind(this))
```

Comment:
Instead of creating a `setStateCallback` method on the class, I'd suggest just using an inline function.

```js
this.computeState(null, (err, state) => {
  if (err) throw err
  this.setState(state)
})
```